packages/framework/store/src/yjs/awareness.ts

```diff
@@ -96,9 +103,14 @@ export class AwarenessStore<
     this.awareness.setLocalStateField('flags', { ...oldFlags, [field]: value });
   }
 
-  getFlag<Key extends keyof Flags>(field: Key): Flags[Key] | undefined {
-    const flags = this.awareness.getLocalState()?.flags ?? {};
-    return flags[field];
+  getFlag$<Key extends keyof Flags>(
+    field: Key
+  ): ReadonlySignal<Flags[Key] | undefined> {
+    return computed(() => this._flags.value[field]);
```

Comment:
What about this one? Do we still need this `computed` here?

Reply:
OK, this is also unneeded.